AxolotlKit/Classes/TestUtils/AxolotlInMemoryStore.h

```diff
@@ -0,0 +1,16 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+#import "AxolotlStore.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AxolotlInMemoryStore : NSObject <AxolotlStore>
```

Comment:
I want to use this in some other test suites (notably SMK)

Reply:
TIOLI, but I'd like to converge on a single naming convention for our mocks.  I prefer Mock, but we could also use "Stub", "Fake", etc.  So: `MockAxolotlStore` or `MockProtocolStore`?

Reply:
"Mock" SGTM, done.